docs/developer-guide/serialization.rst

```diff
@@ -20,6 +22,148 @@ For more information on the formats that KSQL supports, see :ref:`ksql_formats`.
 
 KSQL provides some additional configuration that allows serialization to be controlled:
 
+.. _ksql_single_field_wrapping
+
+-------------------------
+Single field (un)wrapping
+-------------------------
+
+.. note:: The ``DELIMITED`` format is not effected by single field unwrapping.
+
+Controlling deserializing of single fields
```

Comment:
```suggestion
Control deserializing of single fields
```

Reply:
I think the original `Controlling` works better as the chapter is about 'controlling' deserialization of single fields...

Reply:
Dat's cool, it's just that the two gerunds sound a bit awkward. :)